portfolio/src/main/webapp/script.js

```diff
@@ -79,28 +79,28 @@ async function isLoggedIn() {
   return status;
 }
 
-/** Creates a chart and adds it to the page. */
+/** Fetches Amazon Stock data and uses it to create a chart. */
 function drawChart() {
-  const data = new google.visualization.DataTable();
-  data.addColumn('string', 'Animal');
-  data.addColumn('number', 'Count');
-  data.addRows([
-    ['Lions', 10],
-    ['Tigers', 5],
-    ['Bears', 15],
-    ['Elephants', 4],
-    ['Snakes', 8]
-  ]);
-  const options = {
-    'title': 'Zoo Animals',
-    'width':500,
-    'height':400
-  };
-  const chart = new google.visualization.PieChart(document.getElementById('chart-container'));
-  chart.draw(data, options);
+  fetch('/amazon-data').then(response => response.json())
+  .then((AmazonData) => {
```

Comment:
nit: this should also be `amazonData`

Reply:
addressed in "fixed case of a in amazonData"